packages/material-ui/src/Grid/Grid.d.ts

```diff
@@ -3,13 +3,13 @@ import { SxProps, SystemProps } from '@material-ui/system';
 import { Theme } from '../styles';
 import { OverridableComponent, OverrideProps } from '../OverridableComponent';
 
-export type GridDirection = 'row' | 'row-reverse' | 'column' | 'column-reverse';
+type GridDirection = 'row' | 'row-reverse' | 'column' | 'column-reverse';
```

Comment:
Do we even need to create these types? Would it make sense to inline them?

Reply:
Since we're not exporting them, inline is better. I only kept the alias for `GridSize` because it's used several times.